src/ARCtrl/ARC.fs

```diff
@@ -287,6 +294,52 @@ type ARC(?isa : ISA.ArcInvestigation, ?cwl : CWL.CWL, ?fs : FileSystem.FileSyste
             | Some (dto,wb) -> Contract.createCreate(fp,dto,DTO.Spreadsheet wb)
             | None -> Contract.createCreate(fp, DTOType.PlainText)
         )
+    /// <summary>
+    /// This function returns the all update Contracts for the current state of the ARC. Only update contracts for those ISA objects that have been changed will be returned. If an ISA object was added to the ARC, instead a write contract for the complete object folder will be returned. 
+    /// 
```

Comment:
Add a specification when an object was changed. (I assume it is when hashcode = 0)